frontend/src/pages/TrainingReports/components/EventCard.js

```diff
@@ -11,14 +11,20 @@ import { checkForDate } from '../../../utils';
 import ExpanderButton from '../../../components/ExpanderButton';
 import SessionCard from './SessionCard';
 import './EventCard.scss';
+import Modal from '../../../components/Modal';
+import isAdmin from '../../../permissions';
 
 function EventCard({
   event,
   onRemoveSession,
+  onDeleteEvent,
+  index,
```

Comment:
I'm sure this is the name I used when I sent you the snippet to get this to work, but do you think we could change this prop to be called "zIndex"?

Reply:
done.